mobile/src/main/java/com/gelakinetic/mtgfam/fragments/TradeFragment.java

```diff
@@ -362,9 +367,9 @@ public void showDialog(final int id, final int sideForDialog, final int position
     /**
      * Sort the trades
      */
-    private void sortTrades(String sortOrder) {
+    private void sortTrades(String sortOrder, boolean dontSort) {
         /* If no sort type specified, return */
-        TradeComparator tradeComparator = new TradeComparator(sortOrder);
+        TradeComparator tradeComparator = new TradeComparator(sortOrder, dontSort);
         Collections.sort(mLeftList, tradeComparator);
         Collections.sort(mRightList, tradeComparator);
```

Comment:
I would prefer making a shallow copy of the mLeftList for display and sorting. From a design perspective, the conceptual mtg card does not have a "order added index", but a trade/wishlist does have an order. So in that sense would such a design be a better fit.